cPP_DSC.adoc

```diff
@@ -2383,7 +2397,9 @@ Testing is specified for functional aspects of the system as well as aspects tha
 
 ==== Independent Testing - Conformance (ATE_IND.1)
 :xrefstyle: short
+
 Testing is performed to confirm the functionality described in the TSS as well as the operational guidance (includes "evaluated configuration" instructions). The focus of the testing is to confirm that the requirements specified in <<Security Functional Requirements>> are being met. The Evaluation Activities in the SD identify the specific testing activities necessary to verify compliance with the SFRs. The evaluator produces a test report documenting the plan for and results of testing, as well as coverage arguments focused on the platform/TOE combinations that are claiming conformance to this cPP.
+
 :xrefstyle: full
```

Comment:
This `:xrefstyle: full` was visible in the PDF. I hope the spacing will fix this, but I did not test it.

Reply:
#261